### PR TITLE
Scrollbar styles

### LINKED ISSUE #1145: [0.17] [WEB, FF, Brave] scroll appears in the confirmation modal on a screen with a certain resolution
Very-very low priority

Environment: Win 10, Firefox and Brave browsers
Screen resolution: 1536x864

1. Open Swap form
2. Connect to a wallet
3. Specify From/To tokens, From amount
4. Open Confirmation modal
5. Wait till prices are updated

**AR**: scroll appears in the confirmation modal 

**ER**: no scroll

### PR DESCRIPTION
**Pull request by alongoni**
# Summary
Fixes #1145
**Confirmation Modal:**
image

**Select a token modal:**
Before/After

**Styles on main scrollbar:**
Before:
image
After:
image

  # To Test

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @alongoni , great!

A couple of issues I have faced: 
1. I'm able to see 2 scroll bars when I open the menu in a tablet mode.
double

 In order to reproduce this issue, please expand Swap form as much as you can (specify a token pair with amounts, add recipient to the trade, etc). Then reduce the screen to the tablet mode and open the menu

2. New Main scrollbar looks good in Chrome, but still it is not changed in FF. Will we be able to fix this as well?
image

Thanks!

**Comment by alongoni** [comment]
> Hey @alongoni , great!
> 
> A couple of issues I have faced:
> 
> 1. I'm able to see 2 scroll bars when I open the menu in a tablet mode.
>    double
> 
> In order to reproduce this issue, please expand Swap form as much as you can (specify a token pair with amounts, add recipient to the trade, etc). Then reduce the screen to the tablet mode and open the menu
> 
> 2. New Main scrollbar looks good in Chrome, but still it is not changed in FF. Will we be able to fix this as well?
>    image
> 
> Thanks!

Hey @elena-zh 
1. I couldn't reproduce this issue with the double scroll (Chrome and FF). Can you give some context to reproduce it?
Also I found another issue in medium devices. It seems that sometimes when we open the menu some options are cropped.
https://www.loom.com/share/70ff6c55188a4cd699375f621e14849b

2. Done! Now works fine on Firefox.

**Comment by elena-zh** [comment]
Hey @alongoni , I'm also no longer possible to reproduce the issue with a double scroll. 

Scrollbar in FF looks' good to me now! 

However, I have noticed another issue: when open a token list, and it is loading, you will see this (see the screenshot) for 2-3 sec
loading

Could we fix this?

Also, can we enhance this scrollbar?
this

File: src/custom/components/SearchModal/CommonBases/index.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const AutoColumn = styled(AutoColumnUni)`
 
 export const MobileWrapper = styled(AutoColumn)`
   ${({ theme }) => theme.mediaWidth.upToSmall`
-    isplay: none;
```
Comment (alongoni) [inline src/custom/components/SearchModal/CommonBases/index.ts:33]: Actually in mobile devices we can see the common tokens row. 
image

What should we do with this? 
If we fix this typo with `display: none;` , the common token list will disappear:
image

I believe we can keep it.
cc @elena-zh @alfetopito @michelbio

**Review by elena-zh — APPROVED** [review]
LGTM!

**Pull request merged**